### ISSUE TITLE
HTML Templates?

### ISSUE THREAD

**Issue by Outstep** [privacore/open-source-search-engine#42]
Hello,

I was looking over the FindX (Gigablast) code and site to get a feel for the search engine and was trying to locate the HTML template for the main and results pages, but could not find any.

Could you please tell me which files house these templates?

Thanks
Lonnie

**Comment by br-privacore** [comment]
The original Gigablast developer decided to hard-code the HTML into the search engine code instead of using templates, so its scattered around in e.g. PageResults.cpp

If you are considering using it, I'd recommend using the search engine's JSON API and develop your own frontend.

**Comment by Outstep** [comment]
Thanks for getting back to me on this.

I had been playing around with another type of search engine that used templates and really liked the way that FindX displays its results as well as the other pages like "images", "video" and "maps"

Was hoping to get some solid html pages so that I could see how some of that was done.  I had looked at the page source for some searches in a browser, but the pages are really hard to follow as they are not formatted to easily see what is happening.

Anyway, I'll look around more but must say that you have done some nice work on cleaning up the old Gigabase code. I still like it as I do a little with CPP and like that language more than most.

Wish you the greatest success.
Cheers,
Lonnie
